dbt-cta/twilio/models/0_ctes/available_phone_numbers_local_ab1.sql

```diff
@@ -0,0 +1,32 @@
+{{ config(
+    cluster_by = "_airbyte_emitted_at",
+    partition_by = {"field": "_airbyte_emitted_at", "data_type": "timestamp", "granularity": "day"},
+    unique_key = '_airbyte_ab_id',
+    tags = [ "top-level-intermediate" ]
+) }}
+-- SQL model to parse JSON blob stored in a single column and extract into separated field columns as described by the JSON Schema
+-- depends_on: {{ source('cta', '_airbyte_raw_available_phone_numbers_local') }}
+select
+    {{ json_extract_scalar('_airbyte_data', ['beta'], ['beta']) }} as beta,
+    {{ json_extract_scalar('_airbyte_data', ['lata'], ['lata']) }} as lata,
+    {{ json_extract_scalar('_airbyte_data', ['region'], ['region']) }} as region,
+    {{ json_extract_scalar('_airbyte_data', ['latitude'], ['latitude']) }} as latitude,
+    {{ json_extract_scalar('_airbyte_data', ['locality'], ['locality']) }} as locality,
+    {{ json_extract_scalar('_airbyte_data', ['longitude'], ['longitude']) }} as longitude,
+    {{ json_extract_scalar('_airbyte_data', ['iso_country'], ['iso_country']) }} as iso_country,
+    {{ json_extract_scalar('_airbyte_data', ['postal_code'], ['postal_code']) }} as postal_code,
+    {{ json_extract_scalar('_airbyte_data', ['rate_center'], ['rate_center']) }} as rate_center,
+    {{ json_extract('table_alias', '_airbyte_data', ['capabilities'], ['capabilities']) }} as capabilities,
+    {{ json_extract_scalar('_airbyte_data', ['capabilities','MMS'], ['capabilities','MMS']) }} as capabilities_MMS,
```

Comment:
oh oh it turns out unnesting nested json fields isnt too hard, just have to create a list with all the levels. So for this case the json looks like 
```json
{
  "capabilities":
      "MMS": true,
      "SMS": true,
      "voice": true
```
So if you want each individual field you gotta give the path to it as a list with the `json_extract_scalar` macro.